cmd/manager/main.go

```diff
@@ -145,13 +145,18 @@ func main() {
 	// 	log.Info(err.Error())
 	// }
 
-	// Set up WebHook listener
+	// Set up WebHook listener and healthz endpoint
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/webhook/", func(w http.ResponseWriter, r *http.Request) {
 		handler.WebhookHandler(w, r, gitopsconfig.NewReconciler(mgr))
 	})
 
+	mux.HandleFunc("/healthz", func(w http.ResponseWriter, _ *http.Request) {
```

Comment:
*Nice,* the `_` is a fine touch!

Also, huge thanks for including the "why" in the commit message. Highly appreciated!